Makefile

```diff
@@ -49,16 +49,14 @@ endif
 all: clean print_info
 	@echo ----------------------------------------------------------
 	@echo "Packaging module."
-	tar --transform "s|module|$(PACKAGE_NAME)-$(PACKAGE_VERSION)|g" \
-		--exclude=./.gitattributes \
+	tar --exclude=./.gitattributes \
 		--exclude=./.travis.yml \
 		--exclude=CMSIS/index.html \
 		--exclude=CMSIS/Documentation \
 		--exclude=CMSIS/Pack \
 		--exclude=CMSIS/Utilities \
 		--exclude=CMSIS/DSP_Lib/Examples \
 		--exclude=Device/ARM/Documents \
-		--exclude-vcs \
```

Comment:
--exclude-vcs is by essence --exclude=.git but not only ;-)
at least it was doing that when I wrote this makefile